addons/account/models/account_report.py

```diff
@@ -140,6 +140,18 @@ def _validate_root_report_id(self):
             if report.root_report_id.root_report_id:
                 raise ValidationError(_("Only a report without a root report of its own can be selected as root report."))
 
+    @api.constrains('line_ids')
+    def _validate_parent_sequence(self):
+        parent = self.env['account.report.line']
```

Comment:
```suggestion
        previous_lines = self.env['account.report.line']
```

Avoiding confusing variable names always makes for cleaner code ;)